modules/datahandling/messages_test.go

```diff
@@ -0,0 +1,42 @@
+package datahandling
+
+import (
+	"testing"
+)
+
+var testJSON = []byte(
+	"{\"Tag\":12345, " +
+		"\"Resource\":\"Project\", " +
+		"\"Method\":\"Lookup\", " +
+		"\"SenderID\":\"loganga\", " +
+		"\"SenderToken\":\"test\", " +
+		"\"Timestamp\":1460839273, " +
+		"\"Data\":{\"ProjectIds\": [{\"ProjectId\" :12345}]}}")
+
+func TestCreateAbstractRequest(t *testing.T) {
```

Comment:
Do you test any cases where exceptions are thrown? We need to make sure that malformed messages won't crash the server. They should just be logged and ignored.